src/access/examples/PermissiveAccessControlDelegate.cpp

```diff
@@ -27,6 +27,8 @@ using Entry         = chip::Access::AccessControl::Entry;
 using EntryIterator = chip::Access::AccessControl::EntryIterator;
 using Target        = Entry::Target;
 
+bool sDenied = false;
```

Comment:
Er, why?  Wouldn't the right thing to do in a test that wants to change the behavior be to set a _different delegate_?

Reply:
just add new delegate for this acl event test, thanks